### ISSUE TITLE
Port for Telegram

### ISSUE THREAD

**Issue by sivakov512** [arcticicestudio/nord#37]
It would be cool if there was also an implementation for GTK + and Telegram
I see that issue for GTK+ already exists #11

**Comment by sivakov512** [comment]
I am about [this app](https://telegram.org/)

**Comment by arcticicestudio** [comment]
Thanks for your proposal 👍 
I'll add it as a port project to #18 and @-mention you as soon as I start working on it in the backlog ticket so you can track the progress or contribute ideas for this port project.

Please note that it may take some time because there are many tickets with requests and other proposals in the queue. Also take into account that I work as a software developer in the "real world" and I hope you understand that the time to work on my projects may vary depending on my freetime. :octocat:

**Comment by Mauladen** [comment]
Can you please create a repository for Telegram?
The other day I'm going to create the Nord theme for Telegram
How will version 0.1.0 be ready I will do PR

**Comment by Mauladen** [comment]
I literally in couple of days want to be engaged in this port because actively I use Telegram
Do you have any success yet?

**Comment by kutsan** [comment]
@Mauladen Use [this](https://art-2.deviantart.com/art/Arc-2-theme-for-Telegram-654895946) until this issue is solved.

**Comment by Mauladen** [comment]
@kutsan Thanks, but I already have one topic for a Telegram which is based on the Base16.
Based on it, I'll make a theme for Telegram using the Nord color palette.

**Comment by kutsan** [comment]
@Mauladen Share with us if you make it.

**Comment by Mauladen** [comment]
@kutsan Yes, I wanted to support Nord.
It is sad of course that the author from-for the work ceased to do ports actively :(

**Comment by arcticicestudio** [comment]
@Mauladen I'd really like to get all the open issues done regarding Nord, but unfortunately my current free time is really limited due to my job and other important projects (with deadlines). Feel free to create a theme on your own and maybe share it as Gist until there's a official port, e.g. download the Arc theme @kutsan linked and just replace the color values :smile:

**Comment by Mauladen** [comment]
![default](https://user-images.githubusercontent.com/12824589/36953853-e19b09dc-202e-11e8-858e-dd96b1601d70.PNG)

At the moment the port I looks so do not think that you will need to make tiled.png so the background was the colors of the palette Nord

**Comment by Mauladen** [comment]
![default](https://user-images.githubusercontent.com/12824589/36954220-92f2040e-2031-11e8-91ab-279d37a8bdec.PNG)

Waiting for your comments friends

**Comment by kutsan** [comment]
@Mauladen I generally don't like your edit. There is an extra horizontal white line under each message, what's even that? In the left side, some names can't even read because of identical color choices. I think yellow as alternative color doesn't suit well in that aspect.

I think [my modified Arc theme](https://github.com/kutsan/dotfiles/tree/master/.config/telegram) is more Nord-ish.

![image](https://user-images.githubusercontent.com/10108377/36985153-93d6999a-20a7-11e8-9d0e-fc6a62f00fc9.png)

**Comment by Mauladen** [comment]
@kutsan Cool port, but I think it's best to use the blue one in the Nord palette --> #81a1c1

**Comment by cobrabr** [comment]
We can use the [Telegram Theme Maker](https://thememaker.site/) to help @arcticicestudio with a baseline theme that he can work with once he has time. I'm about to give it a try, will post it here later.

**Comment by kutsan** [comment]
@cobrabr I don't think that work out. There are so much value to set there and you can't programatically choose a random appropriate color out of given base theme easily. One should spend time to choose colors one by one IMO.

**Comment by cobrabr** [comment]
@kutsan -- yeah, that's what the Theme Maker does. You can choose the Simple mode if you don't want to go too far (which is what I did) or you can go all out and use the Complete mode and set every nook and cranny. The idea is to help @arcticicestudio so that he doesn't have to do all the work himself.

Here's the theme I'm using right now: [Telegram_NordSimple.zip](https://github.com/arcticicestudio/nord/files/1791838/Telegram_NordSimple.zip)

Like I said, I used the Simple mode, so there are some things I didn't get to change, and I'm not sure I like it 100% but it's a start... :)

**Comment by kutsan** [comment]
@cobrabr You should have added a screenshot, but I did it for you for now.

![image](https://user-images.githubusercontent.com/10108377/37133274-60bac89e-22a3-11e8-9c79-1a4e10ceb85c.png)

**Comment by cobrabr** [comment]
Thanks! I was about to do it, actually. 👍 

It actually looks much better on your screenshot (which is the macOS version) than on my Windows Desktop app...

And as you can see, certain things (like the green "2" for the unread message count) I couldn't control, but the Complete mode is just INSANE, and I just wanted to give it a quick try for now.

**Comment by gilbertw1** [comment]
Hey guys, I just switched over to this theme and decided to make a theme for telegram (as I do with every theme I use). This is what it looks like:

![screenshot](https://github.com/gilbertw1/telegram-nord-theme/raw/master/nord-theme-preview.png)

You can find the repo [here](https://github.com/gilbertw1/telegram-nord-theme) if anyone is interested in using it. I'm not the greatest at creating these themes, so I don't know if it's high enough quality to include here, but feel free to use it if you desire.

**Comment by kutsan** [comment]
@arcticicestudio I think @gilbertw1 did the great job, I tested his version and it's looking really good as a Nord theme. You should try it.

**Comment by arkag** [comment]
I've also been working on a port project for this theme, based on TeleArc. I don't know how to generate a preview PNG, so let me know if you guys need/want it. I'm making this with the intention of porting it over to Android, too.

**Comment by Ansem93** [comment]
Not made by me, but this theme is great:
https://t.me/addtheme/nord_colors

Wish a theme based on Frost now :(

**Comment by arkag** [comment]
I've got one I'm managing now because the above theme wasn't doing it for me. https://t.me/addtheme/Nordlette
